arduino/cores/status.go

```diff
@@ -129,7 +129,7 @@ func (tdep ToolDependency) extractRelease(sc Packages) (*ToolRelease, error) {
 	}
 	release, exists := tool.Releases[tdep.ToolVersion.String()]
 	if !exists {
-		return nil, errors.New("Release Not Found")
+		return nil, errors.New("release Not Found")
```

Comment:
this one could be all lower case I guess... don't know why it has been written this way...

Reply:
I don't know I try to resolve only the problem that the linter reported me. In any case this is why should not be capitalized.
"Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context"
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
I understand, but since you lower-cased the `R` I think you want to lower case also the `N` and the `F`:
`release Not Found` -> `release not found`

Reply:
Done 1da1d44d4858afc5f12e9b95d5cc2c6fb966a675